pkg/utils/docker.go

```diff
@@ -41,11 +41,19 @@ var baseImageNameArr = [2]string{
 	performanceImageName,
 }
 
+const pfeContainerName = "codewind-pfe"
+const performanceContainerName = "codewind-performance"
+
+var containerImageNames = []string{
```

Comment:
Since we know the number of indexes in this array it is good practice to specify the length. Should be written as `[2]string{foo,bar}`

Reply:
I've re-written it as `[...]` so the compiler calculates the length.
I didn't realise using [] actually declared a slice and this list of names doesn't need modifying.